src/index.html

```diff
@@ -3,144 +3,144 @@
+      href="./icons/font/flaticon.css"
       type="text/css"
     />
-    <link rel="stylesheet" href="./icons/font/flaticon.css" />
     <link
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
       integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA=="
       crossorigin="anonymous"
     />
+    <link rel="stylesheet" href="./css/style.scss" />
     <script
       type="text/javascript"
       src="https://npmcdn.com/parse/dist/parse.min.js"
     ></script>
-    <!-- <link rel="stylesheet" href="./css/popups.css" /> -->
+    <script src="https://unpkg.com/magic-grid/dist/magic-grid.cjs.js"></script>
+
     <title>places-been</title>
   </head>
   <body>
     <!-- N A V B A R  -->
     <nav class="navbar">
-      <div class="container container--space-between">
-        <a href="#" class="navbar-logo">
-          <h1 class="navbar__title">places been</h1>
-          <h2 class="navbar__subtitle">a travel journal</h2>
-        </a>
-        <ul class="navbar__links">
-          <li class="navbar__link">
-            <a href="#">About</a>
-          </li>
-          <li class="navbar__link">
-            <a href="#main-buttons">My cities</a>
-          </li>
-          <li class="navbar__link">
-            <a href="#">Contact</a>
-          </li>
-        </ul>
-      </div>
-      <!-- <ul class="navbar--social container--social">
-        <li class="social__links">
-          <a href="#">Ig</a>
+      <ul class="navbar__links">
+        <li class="navbar__list">
+          <a class="navbar__link" href="#">Hey,</a>
         </li>
-        <li class="social__links">
-          <a href="#">Fb</a>
+        <li class="navbar__list">
+          <a class="navbar__link" href="">Stories,</a>
         </li>
-      </ul> -->
+        <li class="navbar__list">
+          <a class="navbar__link" href="#">About</a>
+        </li>
+      </ul>
     </nav>
-    <!-- H E A D E R -->
-    <header class="header container--main">
-      <img class="header__img" src="./img/header2.jpg" alt="" />
+
+    <!-- H E A D E R  -->
+    <header class="header">
+      <!-- slideshow -->
+      <section class="container--slideshow">
+        <div class="slides">
+          <!-- <div class="slide slide-1">
+            <img src="./img/photo3.jpg" alt="" />
+          </div> -->
+          <div class="slide slide-1">
+            <img src="./img/photo1.jpg" alt="" />
+          </div>
+          <div class="slide slide-2">
+            <img src="./img/photo8.jpg" alt="" />
+          </div>
+          <div class="slide slide-3">
+            <img src="./img/photo25.jpg" alt="" />
+          </div>
+          <div class="slide slide-4">
+            <img src="./img/photo0.jpg" alt="" />
+          </div>
+        </div>
+        <div class="slide-controls">
+          <button class="prev-btn"></button>
+          <button class="next-btn"></button>
```

Comment:
Remember every button must have a a11y label. In this case, without a visible text, it can be `aria-label="next photo"` can do it.